tests/test_cli_socket_autodiscovery.py

```
@@ -80,7 +80,7 @@ def _run(self) -> None:
                 server.settimeout(min(0.2, remaining))
                 try:
                     conn, _ = server.accept()
-                except TimeoutError:
+                except (socket.timeout, TimeoutError):
```

Ruff (0.15.13)

[warning] Replace aliased errors with `TimeoutError`

Replace with builtin `TimeoutError`

(UP041)